src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] fileData = readFromFile(fromFileName);
+        String report = createReport(fileData);
+        writeToFile(toFileName, report);
+    }
+
+    private String[] readFromFile(String fileName) {
+        StringBuilder builder = new StringBuilder();
+        File myFile = new File(fileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(myFile))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                builder.append(line).append(" ");
+                line = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file.", e);
+        }
+        return parseFileData(builder.toString());
+    }
+
+    private String[] parseFileData(String FileData) {
+        if (FileData.length() == 0) {
+            return new String[0];
+        }
+        return FileData.split(" ");
+    }
+
+    private String createReport(String[] data) {
```

Comment:
This method doing 2 things:

calculation suppy sum and buy sum
creating report string
I think it will be better to separate this responsibilities